Makefile

```
@@ -147,7 +164,7 @@ acceptance/tests:
 	acceptance/checks.sh
 
 # Upload release file to GitHub.
-github-release: release
+github-release: create-release-dir release
```

```suggestion
github-release: release
```

`release` seems to depend on `create-release-dir` so I think we don't need to re-add the dependency here